### PR TITLE
Google Books (Independent Publisher)

### PR DESCRIPTION
**Pull request by fordosa90**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by fordosa90** [comment]
image

image

image

**Comment by Amjed-Ayoub** [comment]
Hello @fordosa90,
Thank you very much and congratulations on your new submission. Our Certification Engineers will review it very soon.
Please let us know if you have any questions.
Thank you very much again for working with us.

File: independent-publisher-connectors/Google Books/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,1243 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Google Books",
+        "description": "Search and discover data and details about and within books available in the vast library of Google",
+        "version": "1.0",
+        "contact": {
+            "name": "Fördős András",
+            "email": "fordosa90@gmail.com"
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Google Books/apiDefinition.swagger.json:9]: @fordosa90 please provide a "url" property in the contact section.
"url": "Contact URL",

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @fordosa90,
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. 
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by fordosa90** [comment]
@Amjed-Ayoub thank you for the very quick process, great improvements!
Looking forward for further collaboration!

**Comment by Amjed-Ayoub** [comment]
Hello @fordosa90 ,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**